### ISSUE TITLE
`fitter.get_derived_params()` fails for ELL1/ELL1H models with nonzero EPS1DOT/EPS2DOT

### ISSUE THREAD

**Issue by rossjjennings** [nanograv/PINT#1688]
I've found that, for some of the models we've created while working on IPTA EDR3, `fitter.get_derived_params()` is failing due to what looks like an internal PINT bug. These are ELL1H models, but the key thing seems to be that they use the EPS1/EPS2 orbital parameterization and have nonzero OMDOT (actually, EPS1DOT and EPS2DOT). (Whether the OMDOT in this case is really physical is beside the point here -- PINT should be able to handle models of this sort regardless.)

To reproduce the bug, download [J2317+1439-pintbug.zip](https://github.com/nanograv/PINT/files/13621624/J2317%2B1439-pintbug.zip), extract it to get `J2317+1439-pintbug.par` and `J2317+1439-pintbug.tim`, and run:
```
import pint
from pint.models import get_model
from pint.toa import get_TOAs
from pint.fitter import DownhillGLSFitter
pint.logging.setup()

model = get_model('J2317+1439-pintbug.par')
toas = get_TOAs('J2317+1439-pintbug.tim', model=model)
fitter = DownhillGLSFitter(toas, model)
fitter.get_derived_params()
```
This will give a `TypeError: Unsupported operand type(s) for ufunc add: 'Quantity,NoneType'`, which appears to happen because, somewhere along the line, PINT creates a frozen `OMDOT` parameter with no associated uncertainty information and adds it to the model. `fitter.get_derived_params()` sees that `OMDOT` and tries to calculate some derived quantities from it, but  ends up raising an error because it is looking for uncertainty information that isn't there. This is despite the fact that the original par file doesn't contain `OMDOT` at all, only `EPS1DOT` and `EPS2DOT`, and those both have associated uncertainties.

**Comment by dlakaplan** [comment]
Thanks for the very clear issue.  I think this is fixed by #1689.  But more broadly: should `derived_params` return any MTOT if OMDOT < 0?

**Comment by rossjjennings** [comment]
I don't really have a strong opinion on whether it should return a negative MTOT or just log (or include in the string output) a warning, as long as it doesn't crash.
